rest/access-tokens/sync-example/sync-example.5.x.rb

```diff
@@ -0,0 +1,31 @@
+require 'twilio-ruby'
+
+# Required for any Twilio Access Token
+# To set up environmental variables, see http://twil.io/secure
+account_sid = ENV['TWILIO_ACCOUNT_SID']
+api_key = ENV['TWILIO_API_KEY']
+api_secret = ENV['TWILIO_API_KEY_SECRET']
+twilioSyncService = ENV['TWILIO_SYNC_SERVICE_SID']
+
+
+# Required for Sync
+outgoing_application_sid = 'APxxxxxxxxxxxx'
+identity = 'user'
+
+# Create Sync grant for our token
+grant = Twilio::JWT::AccessToken::SyncGrant.new
+
+# Create a "grant" which enables a client to use Sync as a given user
+grant.serviceSid = twilioSyncService
```

Comment:
the name of the argument is `service_sid`. We can update also `twilio_sync_service` here.